src/node/interfaces.cpp

```diff
@@ -2,47 +2,58 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include <interfaces/node.h>
-
 #include <addrdb.h>
 #include <banman.h>
+#include <boost/signals2/signal.hpp>
```

Comment:
why is this moved?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/20494#discussion_r533155930

> why is this moved?

Not sure. This came from https://github.com/bitcoin/bitcoin/pull/19160 where I was using IWYU a lot. I tried to manually sort system includes below project includes after running it, but I wasn't very diligent. It'd be more convenient if there was just a single include section, or an automatic include sorter